extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/index.tsx

```diff
@@ -26,7 +26,10 @@ function WrappedPanelStudyBrowserTracking({ commandsManager, extensionManager, s
 
   const getStudiesForPatientByMRN = _getStudyForPatientUtility(extensionManager);
   const _getStudiesForPatientByMRN = getStudiesForPatientByMRN.bind(null, dataSource);
-  const _getImageSrcFromImageId = _createGetImageSrcFromImageIdFn(extensionManager);
+  const _getImageSrcFromImageId = useCallback(
```

Comment:
what about regular PanelStudyBrowser (non tracking)

Reply:
Yes I am handling that in the  changes in this PR in `WrappedPanelStudyBrowser.tsx`. Is there something else I have to change too?